packages/core/test-helpers/core-test-helpers-kbn-server/src/create_serverless_root.ts

```diff
@@ -70,7 +76,10 @@ function createServerlessES() {
       await es.runServerless({
         basePath: Path.join(REPO_ROOT, '.es/es_test_serverless'),
       });
+      // runServerless doesn't wait until the nodes are up
+      await waitUntilClusterReady(getServerlessESClient());
       return {
+        getClient: getServerlessESClient,
```

Comment:
Same, the serverless ES test harness did not expose a client as the traditional one was doing. I'm just creating a default client to localhost:9200 for now, given that apparently the ports aren't configurable when calling `es.runServerless` yet. We will eventually need to adapt this if the ports become options later.